app/views/templates/cover_letter_template.scala.html

```diff
@@ -496,40 +490,97 @@ <h2 class="heading-medium margin-top-small no-margin-bottom print-header__headin
+                <p class="govuk-body-s">
+                    @Html(messages("pdf.ruling.section.askForReview.solsTeam.emailText"))
+                </p>
+                <p class="govuk-body-s">
+                    @Html(messages("pdf.application.footer.solicitors.office.and.legal.services.address"))
+                </p>
+                <h4 class="govuk-heading-s">@messages("pdf.ruling.section.appealForIndependentTribunal.title")</h4>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealForIndependentTribunal.appealImmediately")
+                </p>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealForIndependentTribunal.courtsAndTribunals")
+                </p>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements")
+                </p>
+                <ul class="govuk-list govuk-list--bullet govuk-body-s">
+                    <li>@messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements.1")</li>
+                    <li>@messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements.2")</li>
+                </ul>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealForIndependentTribunal.relevantInfo")
+                </p>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealForIndependentTribunal.tribunalsInfo")
+                </p>
+                <p class="govuk-body-s">
+                    @Html(messages("pdf.ruling.section.appealForIndependentTribunal.page"))
+                </p>
+                <h4 class="govuk-heading-s">@messages("pdf.ruling.section.appealsAndReviews.title")</h4>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealsAndReviews.moreInfo")
+                </p>
+                <p class="govuk-body-s">
+                    @Html(messages("pdf.ruling.section.appealsAndReviews.page"))
+                </p>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealsAndReviews.copy")
+                </p>
+                <h4 class="govuk-heading-s">@messages("pdf.ruling.section.communicatingByEmail.title")</h4>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.communicatingByEmail.tellUs")
+                </p>
+                <p class="govuk-body-s">
+                    @Html(messages("pdf.ruling.section.communicatingByEmail.page"))
                 </p>
                 <p class="govuk-body-s">
-                    @c.assignee.map(_.name).getOrElse("")
+                    @messages("pdf.ruling.section.communicatingByEmail.copy")
                 </p>
+                <h4 class="govuk-heading-s">@messages("pdf.ruling.section.acknowledgement.title")</h4>
```

Comment:
`govuk-heading-s` makes the h4s a bit smaller than the paragraphs which is why I think the `heading-small` styling was created in this file. Please use the class `heading-small` instead, thanks